src/CTA.WebForms2Blazor/Services/WorkspaceManagerService.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Text;
+
+namespace CTA.WebForms2Blazor.Services
+{
+    public class WorkspaceManagerService
+    {
+        private const string WORKSPACE_DUPLICATE_ERROR = "Attempted to create a new Blazor workspace when one already exists";
+        private const string WORKSPACE_MISSING_ERROR_TEMPLATE = "Workspace {0} operation attempted, but no workspace exists";
+        private const string WORKSPACE_UPDATE_FAILED_ERROR_TEMPLATE = "Workspace {0} operation attempted, but workspace failed to apply changes";
```

Comment:
nit: use pascal case for constants to remain consistent with rest of projects in CTA